### PR TITLE
Add permissions for /find and /send

### PR DESCRIPTION
**Pull request by Raraph84**
All is in the title ^^

### CODE REVIEW

**Comment by Janmm14** [comment]
I do **not** think that the find command should be available to everyone by default!

File: proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java [hunk 1]
```diff
@@ -91,11 +91,11 @@ public void load()
         {
             set( "permissions.default", Arrays.asList( new String[]
             {
-                "bungeecord.command.server", "bungeecord.command.list"
+                "bungeecord.command.server", "bungeecord.command.list", "bungeecord.command.find"
```
Comment (Janmm14) [inline proxy/src/main/java/net/md_5/bungee/conf/YamlConfig.java:94]: I do not think that the find command should be available to everyone by default!

**Comment by Raraph84** [comment]
I think this can be merged so

**Comment by Janmm14** [comment]
The find command is not suitable for everyone as it does expose people on restricted servers the command sender has no access to, unlike, for example, command list.

**Comment by Raraph84** [comment]
The command list is showing players on restricted servers and is accessible by default so I don't see any problem
If the owner wants to remove ability to find players, he has to remove permission to list and find command

**Comment by Raraph84** [comment]
Fixed

**Comment by Raraph84** [comment]
Can this be merged then?

**Pull request merged**